css/index.css

```diff
@@ -0,0 +1,312 @@
+    flex-direction: column;
+    align-items: center;
+    margin: 0 auto;
+    overflow: hidden;
+    width: 65%;
+    max-width: 1300px;
+  }
+  
+  input[type="file"] {
+      opacity: 0;
+      overflow: hidden;
+      position: absolute;
+      z-index: -1;
+  }
+    input[type="file"] + label {
+      font-size: 1.5rem;
+      font-weight: 700;
+      color: #f8f8f8;
+      background-color: #1F0C7B;
+      border-radius: .5em;
+      display: inline-block;
+      cursor: pointer;
+      padding: .5em;
+      transition: opacity 0.3s ease-out, outline 0.3s ease-out;
+    }
+    input[type="file"]:focus + label,
+    input[type="file"] + label:hover {
+      opacity: 0.7;
+      outline: 1px dotted;
+      outline: -webkit-focus-ring-color auto 3px;
+    }
+  .file-upload {
+    text-align: center;
+    display: block;
+    margin-top: 1em;
+  }
+    .file-upload > label,
+    .file-upload > h2,
+    .file-upload > input {
+      display: inline-block;
+    }
+    .file-upload > h2 {
+      margin: 0 7px;
+    }
+    .file-upload i {
+      padding-right: 0.4em;
+    }
+  
+  .content-section {
+    text-align: center;
+    width: 100%;
+    background-color: #DCE4E7;
+    padding: 4em;
+    margin: 3em 0;
+  }
+    .content-section > h2 {
+      margin-bottom: 3em;
+      font-size: 2rem;
+    }
+    .content-section > p {
+      width: 60%;
+      margin: 2em auto;
+      font-size: 1.5rem;
+      text-align: justify;
+    }
+    @media screen and (max-width: 640px) {
+      .content-section {
+        margin: 1em auto;
+        padding: 2em;
+      }
+      .content-section > p {
+        width: 100%;
+        font-size: 1rem;
+        text-align: left;
+      }
+      .content-section > h2 {
+        margin-bottom: 1em;
+      }
+    }
+  
+  .demo-imgs {
+    margin: 0 auto;
+  }
+    .demo-imgs > img {
+      height: 17em;
+      max-width: 45%;
+      border-radius: 1em;
+      display: inline-block;
+      border: 1px solid #F2F2F2;
+    }
+    .demo-imgs > img + img {
+      margin-left: 2em;
+    }
+    @media screen and (max-width: 1024px) {
+      .demo-imgs > img {
+        max-width: 100%;
+        display: block;
+        margin: 2em auto;
+      }
+      .demo-imgs > img + img {
+        margin-left: auto;
+      }
+    }
+    @media screen and (max-width: 640px) {
+      .demo-imgs > img {
+        height: auto;
+      }
+    }
+  
+#file-error, #form-error {
+    color: #C62828;
+  }
+  #file-error {
+    margin-top: 0.5em;
+    font-size: 1em;
+  }
+  input[type="text"].error {
+    border-color: #C62828;
+  }
+  input[type="file"] + label.error {
+    border: 3px solid #c62828;
+    background-color: #EF9A9A;
+    color: #303438;
+  }
+  #forms-list > li.error {
+    border: 3px solid #c62828;
+    margin: 2px;
+  }
+  
+  footer {
+    text-align: center;
+    margin-bottom: 2em;
+    font-size: 1em;
+  }
+  
+  .screen-reader {
+      position: absolute !important;
+      clip: rect(1px, 1px, 1px, 1px);
+  }
+  
```

Comment:
💨 Looks like there may be whitespace on 312?

Reply:
Regarding this whitespace and the EOF comments above, wondering if our linters can help us standardize on this. Does prettier have options to handle this for us? I have no strong opinions either way about EOF newlines or whitespace, but prefer to keep a standard to clean up the diffs.